### PR TITLE
[wip] Create generic components and refactor css

### PR DESCRIPTION
**Pull request by RikFrieling**

### CODE REVIEW

File: components/team-members-overview/TeamMembersOverview.jsx [hunk 1]
```diff
@@ -61,32 +62,18 @@ class TeamMembersOverview extends React.Component {
                     New Humanoid
                 </button>
               </Link>
-              <select
-                className="team-member-overview__button-sort"
-                name="sortBy"
+              <SelectBox
+                options={['First Name', 'Last Name']}
+                optionsValues={['firstName', 'lastName']}
```
Comment (boris-arkenaar) [inline components/team-members-overview/TeamMembersOverview.jsx]: I would call it `optionValues`.

File: shared/components/search-field/SearchField.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import './search-field.scss';
+
+const SearchField = () => (
+  <div className="search-field__wrapper">
```
Comment (boris-arkenaar) [inline shared/components/search-field/SearchField.jsx]: This is your search field, it's the root element, it's your 'block'; use the class `search-field`.
Below, the `input` is a child element, it's more specific, it's an 'element'; use a css class name like `search-field__input`.

File: shared/components/search-field/SearchField.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <input
+      className="search-field"
+      id="search-field"
+      name="searchField"
```
Comment (boris-arkenaar) [inline shared/components/search-field/SearchField.jsx]: Do you use the `id` and the `name`?

File: shared/components/search-field/search-field.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
+@import "../../styles/main";
+
+.search-field__wrapper {
```
Comment (boris-arkenaar) [inline shared/components/search-field/search-field.scss]: Nest 'elements' inside their 'block'.

File: shared/components/select-box/SelectBox.jsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './select-box.scss';
+
+const SelectBox = ({
+  optionsValues, options, onChange, value
+}) => {
+  const optionsList = options.map((option, index) => (
```
Comment (boris-arkenaar) [inline shared/components/select-box/SelectBox.jsx]: Like this, I would place this `const` outside the render function.
However, I would just place this piece of template within the other one.

File: shared/components/select-box/SelectBox.jsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    <option
+      key={`${index + option}`}
+      value={optionsValues[index]}
+    >{option}
```
Comment (boris-arkenaar) [inline shared/components/select-box/SelectBox.jsx]: Newline:
```
>
  {option}
</option>
```

File: shared/components/select-box/SelectBox.jsx [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    </option>
+  ));
+
+  return (
+    <div className="select-box__wrapper">
```
Comment (boris-arkenaar) [inline shared/components/select-box/SelectBox.jsx]: Same goes for these class names as for the other component.

File: components/team-members-overview/TeamMembersOverview.jsx (hunk 1 above)
Comment (RikFrieling) [reply components/team-members-overview/TeamMembersOverview.jsx]: implemented in pr 45

**Comment by RikFrieling** [comment]
Changes requested by Boris implemented in PR #45 .

**Pull request merged**